Makefile

```diff
@@ -1253,6 +1253,23 @@ fmt.h str.h scan.h open.h error.h getln.h exit.h auto_break.h auto_qmail.h \
 auto_users.h byte.h
 	./compile qmail-pw2u.c
 
+qmail-qfilter: \
+load qmail-qfilter.o
+	./load qmail-qfilter
+
+qmail-qfilter.0: \
+qmail-qfilter.1
+
+qmail-qfilter.1: \
+qmail-qfilter.9
+	cat qmail-qfilter.9 \
+	| sed s}QMAILHOME}"`head -n 1 conf-qmail`"}g \
+	> qmail-qfilter.1
+
+qmail-qfilter.o: \
+compile qmail-qfilter.c
+	./compile qmail-qfilter.c
```

Comment:
We discussed this in the pub/mob session and the risk is low, If there is an issue, it would likely be a compile-time error on more obscure platforms.